ui/storybook/src/config/addons/happo/register.js

```diff
@@ -0,0 +1,241 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This code is largely a copy of the following file from happo with some
+// adjustments to work for our specific use case (v7 version of storybook) and
+// to more easily support taking pictures of light and dark mode.
+// We may need to keep an eye on the original source to keep things working.
+// Please comment specific areas of customization to make it easier to handle
+// future changes.
+// https://github.com/happo/happo-plugin-storybook/blob/master/src/register.js
```

Comment:
I put "CUSTOMIZATION" in front of a comment in the places I adjusted. I'm going to send a pointer to this PR to happo support when I'm done here to see if they want to build in features to support this sort of stuff natively in the future.

Reply:
Sent happo support an email this morning, so good on that front!